Server/MusiciansAPP/MusiciansAPP.sln

```diff
@@ -0,0 +1,43 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.1.32328.378
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "API", "API\API.csproj", "{810E34D5-DE05-40C3-AB54-89EC7A4979A2}"
```

Comment:
Please use the solution name (i.e. MusiciansAPP) in your project names and namespaces